lib/holdings/callnumber.rb

```diff
@@ -151,6 +151,10 @@ def internet_resource?
       home_location == 'INTERNET'
     end
 
+    def eresv?
```

Comment:
One random thought would be to add this case somehow to `internet_resource?` so that E-RESV items were also considered internet resources.  This would have the benefit of not needing a change in `present?` as well as label the resulting item in the call number browse as `eResource` instead of `(no call number)`.  It could be a slight improvement (but should def. not be considered a blocker)

Reply:
Yeah, I'm still not clear whether they're the same case or just suspiciously similar.